docs/SetupAzureDatabase-1.md

```diff
@@ -0,0 +1,58 @@
+# Set up a database
+
+This guide explains how to set up a database for Scalar DL in Azure and how to configure helm charts for Scalar DL deployment.
+
+## Cosmos DB
+
+In this section, you will set up a Cosmos DB for Scalar DL.
+
+### Requirements
+
+* You must create a Cosmos DB account with `Core(SQL)` API.
+* You must change Cosmos DB `Default consistency` to `Strong`.
+* You must use Capacity mode as `Provisioned throughput`.
+
+### Steps
+
+* Create an Azure Cosmos DB account on the basis of [Azure official guide](https://docs.microsoft.com/en-us/azure/cosmos-db/create-cosmosdb-resources-portal#create-an-azure-cosmos-db-account) with the above requirements.
+
+### Configure Scalar DL 
+
+To apply the Scalar DL schema to Cosmos DB, update the following configuration in [schema-loading-custom-values](https://github.com/scalar-labs/scalar-kubernetes/blob/master/conf/schema-loading-custom-values.yaml)
```

Comment:
```suggestion
To create Scalar DL schema in Cosmos DB, update the following configuration in [schema-loading-custom-values](https://github.com/scalar-labs/scalar-kubernetes/blob/master/conf/schema-loading-custom-values.yaml)
```

Reply:
Thank you for the suggestion, fixed in 051af67